src/bun.js/node/types.zig

```diff
@@ -865,9 +865,13 @@ pub const PathLike = union(enum) {
 };
 
 pub const Valid = struct {
-    pub fn fileDescriptor(fd: i64, ctx: JSC.C.JSContextRef, exception: JSC.C.ExceptionRef) bool {
+    pub fn fileDescriptor(fd: i64, ctx: JSC.C.JSContextRef, syscall: ?Syscall.Tag, exception: JSC.C.ExceptionRef) bool {
```

Comment:
It would be better to avoid using JSC.C.ExceptionRef instead of passing the syscall through